### ISSUE TITLE
how would this look like in a NUXT project

### ISSUE THREAD

**Issue by dohomi** [PierBover/vuex-apollo-example-project#1]
Hi @PierBover 

I'm very interested in this example. Do you have an idea how to set this up in NUXT as well? I am facing the problem that I don't have the correct initial state on client-server hydration.

Thanks!

**Comment by PierBover** [comment]
I have never used Nuxt, but [from this example](https://nuxtjs.org/examples/vuex-store) it seems there would be absolutely no difference.

You'd initialise Apollo Client somewhere, then import it in your store, then create actions that use Apollo client to query, mutate, subscribe, and unsubscribe.

**Comment by dohomi** [comment]
Yes, maybe if I'd use it in the fetch() method on NUXT I could try to avoid this issue. Otherwise I need a way how to set the `initialState`inside of the apollo-client on client side

**Comment by PierBover** [comment]
Can you show me an example project?

**Comment by dohomi** [comment]
@PierBover currently my project is on hold due to the fact that in NUXT the alpha branch is not documented yet and there are many changes. I will work on this as soon a changelog of 0.x => 1.x exist
